src/components/header/drawer/DrawerSections.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { FC } from 'react';
+import { useI18n } from '@smg-automotive/i18n-pkg';
+import { Box, GridItem } from '@chakra-ui/react';
```

Comment:
Can you use our component `Box` instead importing it through '@chakra-ui/react'? Thanks